src/store/aggregator/aggregator.sagas.ts

```diff
@@ -121,6 +128,58 @@ export function* fetchPausedTokens() {
   }
 }
 
+export function* fetchIncentive() {
+  try {
+
+    yield* put(aggregatorActions.setIncentivesLoading());
+
+    const rewardManager = yield* select(rewardManagerSelector);
+    const sendAmount = yield* select(sendAmountSelector);
+    const flowState = yield* select(flowStateSelector);
+
+    if (!rewardManager) {
+      throw new Error('Could not find RewardManager contract');
+    }
+
+    const startingTokenAddress = (yield* select(startingTokenAddressSelector)) ?? '';
+    const destinationTokenAddress = (yield* select(destinationTokenAddressSelector)) ?? '';
```

Comment:
```suggestion
    const startingTokenAddress = yield* select(startingTokenAddressSelector);
    const destinationTokenAddress = yield* select(destinationTokenAddressSelector);
```
No need for fallbacks, you check them before usage.